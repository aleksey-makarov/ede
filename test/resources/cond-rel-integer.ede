{% if 2 == 2 %}
2 == 2
{% endif %}
{% if 2 == 3 %}
2 == 3
{% endif %}

{% if 6 /= 7 %}
6 /= 7
{% endif %}
{% if 6 /= 6 %}
6 /= 6
{% endif %}

{% if 2 > 1 %}
2 > 1
{% endif %}
{% if 2 > 2 %}
2 > 2
{% endif %}

{% if 11 >= 10 %}
11 >= 10
{% endif %}
{% if 10 >= 10 %}
10 >= 10
{% endif %}
{% if 10 >= 20 %}
10 >= 20
{% endif %}

{% if 3 < 5 %}
3 < 5
{% endif %}
{% if 3 < 3 %}
3 < 3
{% endif %}

{% if 1 <= 10 %}
1 <= 10
{% endif %}
{% if 10 <= 10 %}
10 <= 10
{% endif %}
{% if 11 <= 10 %}
1 <= 10
{% endif %}
