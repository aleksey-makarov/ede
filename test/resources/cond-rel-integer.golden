2 == 2

6 /= 7

2 > 1

11 >= 10
10 >= 10

3 < 5

1 <= 10
10 <= 10
